Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift

```diff
@@ -0,0 +1,125 @@
+import UIKit
+
+public final class SimplifiedLoginManager {
+    public enum SimplifiedLoginError: Error {
+        case noOtherLoggedInSession
+    }
+    
+    var keychainSessionStorage: KeychainSessionStorage?
+    let client: Client
+    var user: User?
+    
+    // Properties for building SimplifiedLoginViewController
+    let env: ClientConfiguration.Environment
+    let withMFA: MFAType?
+    let loginHint: String?
+    let extraScopeValues: Set<String>
+    let completion: LoginResultHandler
+    var withSSO: Bool = true
+    
+    @available(iOS, obsoleted: 13, message: "This function should not be used in iOS version 13 and above")
+    public init(accessGroup: String,
+                client: Client,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
+                withMFA: MFAType? = nil,
+                loginHint: String? = nil,
+                extraScopeValues: Set<String> = [],
+                completion: @escaping LoginResultHandler) {
+        self.keychainSessionStorage = KeychainSessionStorage(service: Client.keychainServiceName, accessGroup: accessGroup)
+        self.client = client
+        
+        self.env = env
+        self.withMFA = withMFA
+        self.loginHint = loginHint
+        self.extraScopeValues = extraScopeValues
+        self.completion = completion
+    }
+    
+    @available(iOS 13.0, *)
+    public init(accessGroup: String,
+                client: Client,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
+                withMFA: MFAType? = nil,
+                loginHint: String? = nil,
+                extraScopeValues: Set<String> = [],
+                withSSO: Bool = true,
+                completion: @escaping LoginResultHandler) {
+        self.keychainSessionStorage = KeychainSessionStorage(service: Client.keychainServiceName, accessGroup: accessGroup)
+        self.client = client
+        
+        self.env = env
+        self.withMFA = withMFA
+        self.loginHint = loginHint
+        self.extraScopeValues = extraScopeValues
+        self.completion = completion
+        self.withSSO = withSSO
+    }
+    
+    // MARK:
+    
+    public func getSimplifiedLogin(completion: @escaping (Result<UIViewController, Error>) -> Void) throws {
+        let latestUserSession = self.keychainSessionStorage?.getAll()
+            .sorted { $0.updatedAt > $1.updatedAt }
+            .first
+        
+        guard let latestUserSession = latestUserSession else {
+            throw SimplifiedLoginError.noOtherLoggedInSession // TODO: This  could also be a missing entitlement error.
```

Comment:
is it possible to differentiate these errors? print to logs and forget about brands questions? :P

Reply:
Hmm.. 
With the current implementation. If we want to propagate up the error.
Then we need to update interface of the `KeychainSessionStorage.getAll()` and `KeychainStorage.getAll()`.
Will it provide any value to propagate the specific error? We are currently not doing it for other places in the app 🤔

Reply:
So maybe its enough to print log with error message (with this info that it can be also connected with entitlement)

Reply:
Yes I will add that, and rename the public error to  `noLoggedInSessionInSharedKeychain`